swmr_tools/Dask_Example.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Dec  3 17:00:21 2020
+
+@author: eja26438
```

Comment:
Don't leak fedid

Reply:
Dask_Example.py has been removed, it was not meant to be in the final repo. It was just where I was playing around with dask.